install_config/aggregate_logging.adoc

```diff
@@ -148,6 +148,10 @@ $ oadm policy add-cluster-role-to-user rolebinding-reader \
 <1> Use the project you created earlier (for example, *logging*) when specifying
 this service account.
 ====
+ifdef::openshift-enterprise,openshift-origin[]
+. Ensure that Port 9300 is open. By default the Elasticsearch service uses Port 9300 for TCP communication between nodes in a cluster.  
+endif::openshift-enterprise[]
```

Comment:
Should the `openshift-enterprise` go away here? So (from the welcome/index.adoc page):

ifdef::openshift-origin,openshift-enterprise[]
17	as a **cluster administrator** or an **application developer**.
18	endif::[]